PeerioTechnologies/peerio-mobile
# feat: maintenance top drawer for mobile [ch13234]

#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/13234/mobile-wire-top-drawer-to-maintenance-warning#activity=13686

#### Testing instructions  

1. trigger the maintenance window
2. the maintenance top drawer should appear
3. after dismissing top drawer it should not reappear after restart until 24 hours later
4. to check that interval is working, set time on device forward 24 hours.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
